src/Vendr.PaymentProviders.Worldpay/WorldpaySettings.cs

```diff
@@ -1,10 +1,58 @@
 using Vendr.Core.Web.PaymentProviders;
 
-namespace Vendr.PaymentProviders
+namespace Vendr.PaymentProviders.Worldpay
 {
     public class WorldpaySettings
     {
-        [PaymentProviderSetting(Name = "Continue URL", Description = "The URL to continue to after this provider has done processing. eg: /continue/")]
+        [PaymentProviderSetting(Name = "Cancel Url", Description = "The Cancel URL")]
+        public string CancelUrl { get; set; }
+
+        [PaymentProviderSetting(Name = "Error Url", Description = "The Error URL")]
+        public string ErrorUrl { get; set; }
+
+        [PaymentProviderSetting(Name = "Continue Url", Description = "The Continue URL")]
         public string ContinueUrl { get; set; }
+
+        [PaymentProviderSetting(Name = "Install ID", Description = "The installation ID")]
+        public string InstallId { get; set; }
+
+        [PaymentProviderSetting(Name = "Mode", Description = "TEST or LIVE")]
+        public string Mode { get; set; }
+
+        [PaymentProviderSetting(Name = "Test Mode Number", Description = "100 for TEST, 0 for LIVE")]
+        public string TestModeNumber { get; set; }
+
+        [PaymentProviderSetting(Name = "Test URL", Description = "Default: https://secure-test.worldpay.com/wcc/purchase")]
+        public string TestUrl { get; set; }
+
+        [PaymentProviderSetting(Name = "Live URL", Description = "Default: https://secure.worldpay.com/wcc/purchase")]
+        public string LiveUrl { get; set; }
+        
+        [PaymentProviderSetting(Name = "Auth Mode", Description = "A for a full authorisation, or E for a pre-authorisation")]
+        public string AuthMode { get; set; }
+
+        [PaymentProviderSetting(Name = "MD5 Secret", Description = "If enabled enter secret, fields are amount, currency, instId and cartId parameters")]
+        public string Md5Secret { get; set; }
+
+        [PaymentProviderSetting(Name = "Response Password", Description = "If enabled enter response password")]
+        public string ResponsePassword { get; set; }
+
+        [PaymentProviderSetting(Name = "Verbose Logging", Description = "Enable verbose logging")]
+        public bool VerboseLogging { get; set; }
```

Comment:
Given this is a dev setting, I'd probably move this to be last and make it an advanced property https://github.com/vendrhub/vendr-payment-provider-stripe/blob/dev/src/Vendr.PaymentProviders.Stripe/StripeSettingsBase.cs#L84